astropy/cosmology/flrw.py

```diff
@@ -249,6 +250,16 @@ def __init__(self, H0, Om0, Ode0, Tcmb0=0.0*u.K, Neff=3.04, m_nu=0.0*u.eV,
         self._inv_efunc_scalar = self.inv_efunc
         self._inv_efunc_scalar_args = ()
 
+        # vectorized methods, created on the instance
+        # TODO! When quantity-aware vectorization can do this on the class.
+        self._integral_age = np.vectorize(self._integral_age_scalar, cache=True)
+        self._integral_lookback_time = \
+            np.vectorize(self._integral_lookback_time_scalar, cache=True)
+        self._integral_absorption_distance = \
+            np.vectorize(self._integral_absorption_distance_scalar, cache=True)
+        self._integral_comoving_distance_z1z2_vec = \
```

Comment:
Why does this one have the `_vec` suffix?

Reply:
This method is a little more complex.  It’s called in a few places and leads to a circular set of function calls if not named differently. It’s all private methods and when I get Quantity-aware `frompyfunc` going I can revisit this and hopefully clean some stiff up.

Reply:
OK, thanks!